environments/pull-environment.sh

```diff
@@ -25,7 +25,7 @@ AWSCLOUDFORMATIONCONFIG="{\
 \"profileName\":\"$AWS_PROFILE\",\
 \"accessKeyId\":\"$AWS_ACCESS_KEY_ID\",\
 \"secretAccessKey\":\"$AWS_SECRET_ACCESS_KEY\",\
-\"region\":\"us-east-1\"\
+\"region\":\"us-east-2\"\
```

Comment:
New environments are now in `us-east-2`. We will incrementally migrate all e2e resources to it, starting with `geo` in this PR.

Reply:
Do we have any documentation which account and region contents what? Might be helpful to refer in the PR description?

Reply:
For my knowledge, why we put it in `CMH`?

Reply:
@0618 I tried to keep internal details out of pr description, but I realize this was really unclear. I'll share that async.

Reply:
@zchenwei No strong reason, but mainly to keep resources in the same region as other aws resources we own.